src/modules/UI/Settings/spendingLimits/SpendingLimitsReducer.js

```diff
@@ -23,7 +23,7 @@ export const newSpendingLimits = (spendingLimits: SpendingLimits) => ({
 // REDUCERS
 export const initialState = {
   transaction: {
-    isEnabled: false,
+    isEnabled: true,
```

Comment:
Is there a reason we are setting the default to true / enabled?

Reply:
it should be false